lib/engine/game/g_18_usa/game.rb

```diff
@@ -630,20 +630,78 @@ def operating_round(round_num)
             G1817::Step::CashCrisis,
             G18USA::Step::ObsoleteTrain,
             G18USA::Step::Loan,
+            G18USA::Step::ScrapTrain,
             G18USA::Step::SpecialTrack,
             G18USA::Step::SpecialToken,
             G18USA::Step::SpecialBuyTrain,
             G18USA::Step::Assign,
             G18USA::Step::Track,
             G18USA::Step::DenverTrack,
             G18USA::Step::Token,
+            G18USA::Step::BuyPullman,
             G18USA::Step::Route,
             G18USA::Step::Dividend,
-            Engine::Step::DiscardTrain,
+            G18USA::Step::DiscardTrain,
             G18USA::Step::BuyTrain,
           ], round_num: round_num)
         end
 
+        def pullman_scrap_value
+          50
+        end
+
+        def scrap_info
+          "Scrap Pullman for #{format_currency(pullman_scrap_value)}"
+        end
+
+        def scrap_button_text
+          'Scrap Pullman'
+        end
+
+        def use_compact_scrap_trains_view
+          true
+        end
+
+        def use_1840_style_merger_round_scrap_trains?
+          false
+        end
```

Comment:
The base class default should be false and then defined in 1840.

The name could also be like use_operating_round_for_merger_round_train_actions, so its more generic.

Reply:
Looks like I've given two naming ideas. Names are hard. :-)